chaste_codegen/chaste_model.py

```diff
@@ -205,10 +205,10 @@ def __init__(self, model, file_name, **kwargs):
              'derived_quantities': self._format_derived_quant(),
              'derived_quantity_equations': self._format_derived_quant_eqs()}
 
-    def _get_equations_for(self, variables, recurse=True):
+    def _get_equations_for(self, variables, recurse=True, filter_modifiable_parameters_lhs=True):
```

Comment:
Maybe document function parameters?

Also, if subclasses are intended to use a method, perhaps it shouldn't start with an underscore?